client-app/src/Components/AddNewStatus.tsx

```diff
@@ -0,0 +1,205 @@
+import React, { useState, useEffect, ChangeEvent, FormEvent } from 'react';
+import axios from 'axios';
+import { useParams } from 'react-router-dom';
+import { useNavigate } from 'react-router-dom';
+import ItemStatus from '../constants/Enums';
+import '../css/DonorForm.css';
+
+interface FormData {
+    statusType: string; 
+    dateModified: string;
+    donatedItemId: string;
+}
+
+interface FormErrors {
+    [key: string]: string;
+}
+
+interface Option {
+    value: string;
+    label: string;
+    id?: number;
+}
+
+const StatusUpdate: React.FC = () => {
```

Comment:
@truffer11 please change the component as AddNewStatus. It is trivial change, just to keep variable names relevant to the context